query/src/main/scala/filodb/query/PromQueryResponse.scala

```diff
@@ -68,4 +68,16 @@ final case class LabelCardinalitySampl(metric: Map[String, String],
                                        cardinality: Seq[Map[String, String]]) extends MetadataSampl
 
 final case class TsCardinalitiesSampl(group: Map[String, String],
-                                      cardinality: Map[String, Int]) extends MetadataSampl
\ No newline at end of file
+                                      cardinality: Map[String, Int]) extends MetadataSampl
+
+/**
+ * @param group map of shardKeyPrefix and its values. Key includes - _ws_, _ns_, __name__
+ * @param cardinality map of string and int (cardinality count). Key includes - active, shortTerm, longTerm
+ * @param dataset This is the user specified dataset. Valid values are - raw, aggregated, recordingrules
```

Comment:
I think all dataset names are valid, no?

Reply:
Yes, they are all valid. I meant for a single row, there could be only of the value in that list.

Reply:
Updated comment to be more specific :)